TopQuarkAnalysis/TopEventProducers/src/StEvtSolutionMaker.cc

```diff
@@ -2,17 +2,55 @@
 //
 
 #include <memory>
+#include <string>
+#include <vector>
+#include <iostream>
+#include <fstream>
 
-#include "TopQuarkAnalysis/TopEventProducers/interface/StEvtSolutionMaker.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "AnalysisDataFormats/TopObjects/interface/StEvtSolution.h"
+#include "TopQuarkAnalysis/TopKinFitter/interface/StKinFitter.h"
+//#include "TopQuarkAnalysis/TopJetCombination/interface/TtJetCombinationProbability.h"
+
+class StEvtSolutionMaker : public edm::stream::EDProducer<> {
+public:
+  explicit StEvtSolutionMaker(const edm::ParameterSet&);
+
+  void produce(edm::Event&, const edm::EventSetup&) override;
+
+private:
+  std::unique_ptr<StKinFitter> myKinFitter;
+  //std::vector<TtJetCombinationProbability> jetCombProbs;
```

Comment:
Also this is linked to the TtJetCombinationProbability class, which does not even exist: since they can be easily spotted, the already commented out references to it could be removed, here and below...

Reply:
Removed.